packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -99,7 +121,32 @@ const BuySellModal = ({ table_type, selected_ad, should_show_popup, setShouldSho
         />
     );
     const [is_account_balance_low, setIsAccountBalanceLow] = React.useState(false);
+    const [show_market_rate_change_error_modal, setShowMarketRateChangeErrorModal] = React.useState(false);
+    const [has_rate_changed_recently, setHasRateChangedRecently] = React.useState(false);
     const formik_ref = React.useRef();
+    const MAX_ALLOWED_RATE_CHANGED_WARNING_DELAY = 1000;
+
+    React.useEffect(() => {
+        const disposer = reaction(
```

Comment:
Can we change disposer to a specific name for this reaction?

Reply:
Sure!